modules/api/src/user/interfaces/newUser.interface.ts

```diff
@@ -0,0 +1,10 @@
+
+export interface INewUser {
```

Comment:
Maybe we should move these typed interfaces to a modules/shared/ folder so the front end and back end and use them. Or we could make a way to pull all type info from the swagger generation for the frontend. 
either way.

Reply:
breaks passport 🤷‍♀️